src/app/pages/quickMaps/pages/dietaryChange/components/comparisonCard/scenariosMap/scenariosMap.component.ts

```diff
@@ -98,45 +94,23 @@ export class ScenariosMapComponent implements AfterViewInit {
 
   private initialiseListeners(): void {
```

Comment:
I think you should still have your move and zoom listeners as they were, making it reflect changes straight away.  I just think that after they have called `setMapZoom` or `setMapPosition` they should then call your `alignMaps` function to ensure that everything is aligned after all zooms and movements are done.

Reply:
I had a zoom listener before, something to do with a trigger on animation was causing it so jump around a bit if it wasn't used, that's why the mousedown() listener was being used too. But as I am now using setView() with animation disabled, the move() listener is called when user zooms too, and it is working properly.

Reply:
So are you happy with the mirrored map not aligning until the user stops moving then? I thought that was quite nice.

Reply:
It still mirrors and is more responsive than the current implementation as zoom all pan animations are disabled. You can test the current implementation [here](https://develop--micronutrientsupport-tool.netlify.app/quick-maps/diet/dietary-change?country-id=NGA&mnd-id=C&measure=diet&data-level=country&age-gender-group=all)

Reply:
I've added @bgsandan as a reviewer to give the stamp of approval.

Reply:
Looks good to me :+1: